lib/truncate.js

```diff
@@ -243,3 +267,20 @@ function contextLength (path, opts) {
 
   return opts.truncateStringsAt
 }
+
+function isHeader (path) {
+  return path[0] === 'context' && (path[1] === 'request' || path[1] === 'response') && path[2] === 'headers' && path[3]
+}
+
+function onObjectWithHeaders (value, key, path, isNew) {
+  if (isHeader(path)) return value.toString()
```

Comment:
Why not use `String(value)` like you do in `truncTransaction` and `truncError`?

Reply:
Ah, yeah...good point. Just a second. 😅

Reply:
fixed.